assets/javascripts/omise-payment-form-handler.js

```diff
@@ -255,9 +255,10 @@
 	}
 
 	function initializeSecureCardForm() {
-		if (Boolean(omise_params.secure_form_enabled)) {
+		const omiseCardElement = document.getElementById('omise-card');
```

Comment:
How do we test?

Reply:
I can recreate the issue locally by following the flow described by CS team.

Reply:
But what about unit tests?

Reply:
Not possible to write unit test currently. It is not based on npm.